src/ig/IGExporter.ts

```diff
@@ -183,33 +179,91 @@ export class IGExporter {
+    let generation: ImplementationGuideDefinitionPageGeneration = 'markdown';
+    if (fs.existsSync(inputIndexMarkdownPath)) {
+      fs.copySync(inputIndexMarkdownPath, path.join(igPath, 'input', 'pagecontent', 'index.md'));
+    } else if (fs.existsSync(inputIndexXMLPath)) {
+      fs.copySync(inputIndexXMLPath, path.join(igPath, 'input', 'pagecontent', 'index.xml'));
+      generation = 'html';
     } else {
       outputFileSync(
         path.join(igPath, 'input', 'pagecontent', 'index.md'),
         this.pkg.config.description ?? ''
       );
     }
+
+    // Add user-provided or generated index file to IG definition
+    this.ig.definition.page.page.push({
+      nameUrl: 'index.html',
+      title: this.ig.title,
+      generation
+    });
   }
 
   /**
-   * SUSHI doesn't yet support authors adding additional pages beyond index.md.  If we detect
-   * additional pages, we need to log an error.
+   * Adds additional pages beyond index.md that are defined by the user.
+   * Only add formats that are supported by the IG template
+   * Intro and notes file contents are injected into relevant pages and should not be treated as their own page
+   *
+   * @param igPath {string} - the path where the IG is exported to
    */
-  private checkForOtherPageContent() {
+  private addOtherPageContent(igPath: string) {
     const inputPageContentPath = path.join(this.igDataPath, 'input', 'pagecontent');
     if (fs.existsSync(inputPageContentPath)) {
-      const pages = fs.readdirSync(inputPageContentPath);
-      if (pages.length > 1 || (pages.length === 1 && pages[0] !== 'index.md')) {
-        logger.error('SUSHI does not yet support custom pagecontent other than index.md.', {
+      const pages = fs
+        .readdirSync(inputPageContentPath)
+        .filter(page => page !== 'index.md' && page !== 'index.xml')
+        .sort(); // Sorts alphabetically
+
+      let invalidFileTypeIncluded = false;
+      pages.forEach(page => {
+        // All user defined pages are included in input/pagecontent
+        const pagePath = path.join(this.igDataPath, 'input', 'pagecontent', page);
+        fs.copySync(pagePath, path.join(igPath, 'input', 'pagecontent', page));
+
+        const fileName = page.slice(0, page.lastIndexOf('.'));
+        const fileType = page.slice(page.lastIndexOf('.') + 1);
+        const isSupportedFileType = fileType === 'md' || fileType === 'xml';
+        const isIntroOrNotesFile = fileName.endsWith('-intro') || fileName.endsWith('-notes');
+        if (isSupportedFileType) {
+          // Intro and notes files will be in supported formats but are not separate pages, so they should not be added to IG definition
+          if (!isIntroOrNotesFile) {
+            // Valid page files will be added to the IG definition
+            this.ig.definition.page.page.push({
+              nameUrl: `${fileName}.html`,
+              title: `${fileName}`,
```

Comment:
I noticed that when the file names are lowercase (which is probably often the case), it looks kind of funny in the TOC and on the pages:
<img width="772" alt="image" src="https://user-images.githubusercontent.com/2278253/73975441-51cf9480-48f4-11ea-9399-40437aa039f4.png">
and a page for example:
<img width="749" alt="image" src="https://user-images.githubusercontent.com/2278253/73975512-7a578e80-48f4-11ea-9ccf-3eef00a9389b.png">

I think it might be nice if we tried to be a little smart about the titles.  I just did a quick test w/ some libraries and I think something like this might be nice:
```js
const { titleCase } = require("title-case");
const _ = require('lodash');

[
  'myTestFileWithoutSpaces',
  'MyTestFileWithoutSpaces',
  'my-test-file-with-dashes',
  'my_test_file_with_underscores'
].forEach(fileName => {
  console.log(titleCase(_.words(fileName).join(' ')));
});

// My Test File Without Spaces
// My Test File Without Spaces
// My Test File with Dashes
// My Test File with Underscores
```

Reply:
I like this! I'll add it in now!